pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -105,8 +107,8 @@ type ReconcileWindowsMachineConfig struct {
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileWindowsMachineConfig) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
-	reqLogger.Info("reconciling WindowsMachineConfig")
+	r.log = log.WithValues("namespace", request.Namespace, "name", request.Name)
```

Comment:
I think asserting that this is log line is coming from controller will be difficult considering we're not using any reference to `controller_wmc`. How about we create a sublogger and use it? 

Example: https://github.com/uber-go/zap/blob/master/example_test.go, not sure if this is doable but something to look into.

Reply:
I don't understand what you are asking for or what you mean by the above. Either way this was meant to be a small fix to our poor logging in the operator. Anything above that is beyond the scope of the PR.